tardis/montecarlo/montecarlo_numba/numba_interface.py

```diff
@@ -266,6 +267,7 @@ def set_properties(
             temp_last_interaction_type = np.empty(temp_length, dtype=np.int64)
             temp_last_interaction_in_id = np.empty(temp_length, dtype=np.int64)
             temp_last_interaction_out_id = np.empty(temp_length, dtype=np.int64)
+
```

Comment:
random whitespace?

Reply:
This Whitespace was added so that it could demarcate that the initialisation ends here & value assignment starts afterwards.